### ISSUE TITLE
_request method unchecked response

### ISSUE THREAD

**Issue by scottwthompson** [Lynten/stanford-corenlp#12]
https://github.com/Lynten/stanford-corenlp/blob/master/stanfordcorenlp/corenlp.py#L176

    r_dict = json.loads(r.text)

should be try/except or check r.ok/status.

Method has invoked uncaught exceptions for me

**Comment by Lynten** [comment]
Yes, somtimes it will raise  an error. The wrapper is just a simple parser interacting with the Java web server which is not stable, so I leave the exception to be handled outside by users.
